src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/BeslutteVedtakSteg.kt

```diff
@@ -134,6 +140,19 @@ class BeslutteVedtakSteg(
         taskService.save(ferdigstillGodkjenneVedtakTask)
     }
 
+    private fun validerBrevmottakere(
+        behandlingId: BehandlingId,
+        totrinnskontrollErGodkjent: Boolean,
+    ) {
+        val brevmottakere = brevmottakerService.hentBrevmottakere(behandlingId.id)
+        if (totrinnskontrollErGodkjent && !BrevmottakerAdresseValidering.erBrevmottakereGyldige(brevmottakere)) {
+            throw FunksjonellFeil(
```

Comment:
Burde erBrevmottakereGyldige omdøpes til harBrevmottakereGyldigAddresse?

Når jeg leser funksjonsnavnet erBrevmottakereGyldige så tenker jeg at man sjekker på alt.
Altså at alle felter i BrevmottakerDto blir sjekket. Mens her sjekkes egentlig bare addresse, samt at feilmeldingen påpeker også bare at addresse er feil.


Gitt at erBrevmottakereGyldige hadde kastet feil grunnet feil i type eller navn, så er ikke frontend meldingen så passende. Hvis det ga mening 🤔

Reply:
Ukjent med brevmottaker domenet, men er det mulig å legge til flere brevmottakere?
Bør man da referere til hvilket brevmottaker som har ugyldig adresse, så det blir lettere for SB å rette opp i?

Reply:
Enig med forslaget med nytt navn. Fikser det nå. 😊 

Det er mulig ja å ha flere manuelle brevmottakere og vi har allerede på plass en frontendvalidering for å sjekke at riktig data blir lagt inn på riktig type brevmottaker. Denne valideringen er for de brevmottakerne som allerede hadde blitt lagt til før frontendvalideringen var på plass (i forrige uke), eksempelvis i EØS-saker som kan bli liggende lenge. I fremtiden burde vi ikke komme i denne staten, derfor prioriterer jeg å få dette ut uten å legge på beskrivende feilmelding på hvilken brevmottaker det gjelder.